### PR TITLE
Unittest projects_summary.py

### PR DESCRIPTION
**Pull request by sjanssen2**

### CODE REVIEW

File: knimin/tests/test_auth_handlers.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        port = self.get_http_port()
+        self.assertEqual(response.effective_url, 'http://localhost:%d/' % port)
+
+    def test_post(self):
+        # check if unkown users are recognized
+        response = self.post('/auth/login/',
+                             {'email': 'idontexist',
+                              'password': 'password',
+                              })
+        self.assertIn('>Unknown user<', response.body)
```
Comment (wasade) [inline knimin/tests/test_auth_handlers.py]: if this does not return a non 200 status code, should it?

File: knimin/tests/test_auth_handlers.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+class TestAuthLogoutHandler(TestHandlerBase):
+    def test_get(self):
+        response = self.get('/auth/login/')
+        self.assertEqual(response.code, 200)
+        # make sure redirect happened properly
+        port = self.get_http_port()
+        self.assertEqual(response.effective_url, 'http://localhost:%d/' % port)
+
+        # TODO: how can I check that the cookie has been cleared? Or don't I
```
Comment (wasade) [inline knimin/tests/test_auth_handlers.py]: Could just do it implicitly by trying to access a page which requires auth

File: knimin/tests/test_projects_summary.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+from unittest import main
+
+from knimin import db
+from knimin.tests.tornado_test_base import TestHandlerBase
+
+
+class TestProjectsSummaryHandler(TestHandlerBase):
+    def test_get_not_authed(self):
+        response = self.get('/projects/summary/')
+        # print(response)
```
Comment (wasade) [inline knimin/tests/test_projects_summary.py]: remove please

File: knimin/tests/test_auth_handlers.py (hunk 1 above)
Comment (sjanssen2) [inline knimin/tests/test_auth_handlers.py]: That's what I would expect, but it doesn't. The return code is always "200", regardless of the success of login. I think we should change this later on to a more speaking return code, but for the current unit tests I have to work with whatever is implemented.

File: knimin/tests/test_auth_handlers.py (hunk 2 above)
Comment (sjanssen2) [inline knimin/tests/test_auth_handlers.py]: That was my initial idea, but isn't there a difference between a logged in user and a stored cookie?

File: knimin/tests/test_projects_summary.py (hunk 3 above)
Comment (sjanssen2) [inline knimin/tests/test_projects_summary.py]: up, will do.

**Comment by sjanssen2** [comment]
How can I update the text for issue #89 ? Could it be, that I don't have the required permissions? I don't see an edit option anywhere.

File: knimin/tests/test_auth_handlers.py (hunk 1 above)
Comment (wasade) [inline knimin/tests/test_auth_handlers.py]: I'd be in favor of a modification to the handler in this respect, what do you think? Or, we can defer to a future issue, but I suspect change is light

**Comment by wasade** [comment]
Should be fixed. Let me know if that didn't work to allow edit permissions.
If it doesn't, I'll just edit

**Comment by wasade** [comment]
Do we care about a cookie if the logged out user isn't able to access
authorized pages?

File: knimin/tests/test_auth_handlers.py (hunk 1 above)
Comment (sjanssen2) [inline knimin/tests/test_auth_handlers.py]: I can't foresee which other parts might break due to this modification. Unittest are exactly for this purpose, don't they? So I would prefer to first complete all unit tests and only then modify the code - but we should raise a new issue here.

**Comment by wasade** [comment]
okay, sounds good

File: knimin/tests/test_auth_handlers.py (hunk 1 above)
Comment (sjanssen2) [inline knimin/tests/test_auth_handlers.py]: opened issue #102

**Comment by sjanssen2** [comment]
stupid me. I added the auth_handler.py to this PR, but it is topic of PR #100  :-/

**Comment by wasade** [comment]
Thanks @sjanssen2!!

**Pull request merged**